deploy-aks/.terraform.lock.hcl

```diff
@@ -0,0 +1,42 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
is this file required?
seems like thats local to your client and probably dont want to include it in the project?

Reply:
Excluded with gitignore file